src/apps/media/src/app/components/MediaDetailsModal.js

```diff
@@ -234,17 +234,22 @@ export const MediaDetailsModal = memo(function MediaDetailsModal(props) {
               value={`${genImageURL()}`}
               sx={{
                 justifyContent: "flex-start",
+                wordBreak: "break-all",
               }}
             />
           </div>
 
           <ul className={styles.info}>
             <li>
-              <span>ZUID:</span>
-              <CopyButton size="small" value={props.file.id} />
+              <p>ZUID:</p>
+              <CopyButton
+                size="small"
+                value={props.file.id}
+                sx={{ width: "fit-content" }}
```

Comment:
Don't believe this width is necessary

Reply:
@agalin920 swapped fit content for align-self: flex-start; if I dont have any styling the button take full width 

<img width="525" alt="Screen Shot 2022-06-22 at 7 37 07 AM" src="https://user-images.githubusercontent.com/22800749/175056684-edb0e84c-c736-42ae-b874-53c3755c8097.png">